bindings/pyroot_experimental/PyROOT/src/PyROOTPythonize.h

```diff
@@ -47,6 +47,8 @@ PyObject *GetSizeOfType(PyObject *self, PyObject *args);
 
 PyObject *MakeNumpyDataFrame(PyObject *self, PyObject *obj);
 
+PyObject *AddTDirectoryFileGetPyz(PyObject *self, PyObject *args);
```

Comment:
Can you place this one up together with the other 3 that are related to TDirectory and children?